### PR TITLE
Adds prometheus exporter to cmd.sh and fixes some bugs

### PR DESCRIPTION
**Pull request by mike1729**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,9 +10,10 @@ Aleph Zero Blockchain python scripts for running TestNet and DevNet.
 - Make sure you have the aleph node binary inside a bin directory (needed for committee key generation step, `cp <...>/aleph-node/target/release/aleph-node bin/`)
 - Install packages needed for orchestrating experiments: GNU parallel, fabric, zip, unzip and Python 3 packages: fabric, boto3, ipython, tqdm, matplotlib, joblib.
 - Then, run `ipython -i shell.py`. This opens a shell with procedures orchestrating experiments.
-  The main procedure is `run_protocol(n_processes, regions, instance_type, volume_size, tag)` that runs `n_processes` spread
+  The main procedure is `setup_nodes(n_processes, regions, instance_type, volume_size, tag)` that prepares the `n_processes` spread
   uniformly across specified `regions` using EC2 machines of `instance_type`. E.g.
   `run_protocol(4, ['eu-west-1'], 't2.micro', 8, 'my_devnet')`.
```
Comment (pmikolajczyk41) [inline README.md:15]:
```suggestion
  `setup_nodes(4, ['eu-west-1'], 't2.micro', 8, 'my_devnet')`.
```

File: utils.py [hunk 2]
```diff
@@ -290,6 +314,9 @@ def bootstrap_chain(account_ids, chain):
     if chain != 'dev':
         chainspec['name'] = 'Aleph Zero Testnet'
 
+    if not os.path.exists('accounts'):
+        os.mkdir('accounts')
+
```
Comment (kostekIV) [inline utils.py:319]: nit: `os.makedirs('accounts', exist_ok=True)` to have one liner :P

File: utils.py [hunk 3]
```diff
@@ -69,6 +69,30 @@ def create_security_group(region_name, ip_list=[], tag=''):
     return sg
 
 
+def allow_all_traffic_in_region(region_name, tag=''):
```
[the remark this answers is not in the document]
Comment (kostekIV) [reply utils.py:72]: oh okay, I was looking at the `shell.py` file.

File: README.md (hunk 1 above)
Comment (mike1729) [reply README.md]: Changed.

File: README.md [hunk 4]
```diff
@@ -10,9 +10,10 @@ Aleph Zero Blockchain python scripts for running TestNet and DevNet.
 - Make sure you have the aleph node binary inside a bin directory (needed for committee key generation step, `cp <...>/aleph-node/target/release/aleph-node bin/`)
 - Install packages needed for orchestrating experiments: GNU parallel, fabric, zip, unzip and Python 3 packages: fabric, boto3, ipython, tqdm, matplotlib, joblib.
 - Then, run `ipython -i shell.py`. This opens a shell with procedures orchestrating experiments.
-  The main procedure is `run_protocol(n_processes, regions, instance_type, volume_size, tag)` that runs `n_processes` spread
+  The main procedure is `setup_nodes(n_processes, regions, instance_type, volume_size, tag)` that prepares the `n_processes` spread
```
[the remark this answers is not in the document]
Comment (mike1729) [reply README.md]: Added.

File: README.md (hunk 1 above)
Comment (pmikolajczyk41) [reply README.md]: my apologies for nitpicking, but as argument for `chain_type` is missing, all the further arguments will be misinterpreted, and the user will be confused with the error :(

**Pull request merged**